serial/serial_resources/dataset.py

```diff
@@ -0,0 +1,24 @@
+import serial
+from ..api_client import APIClient
+
+class Datasets:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get(name, data_type, process_id=None):
+        client = APIClient(serial.api_key, serial.base_url)
+        query_params = {"name": name, "data_type": data_type}
```

Comment:
This typing is so confusing and makes me REALLY want to rewrite all of this on the database side. what's a data_type v.s. type, what has each - makes my head want to explode.

Reply:
my bad... we should probably use `type` here + in zuplo instead of `data_type` then right?

Reply:
No this is intentional - we actually have both types + data_types available - need to update the docs as appropriate.